presentation/src/main/java/com/example/healthysalad/OrderSummaryFragment.kt

```diff
@@ -0,0 +1,81 @@
+package com.example.healthysalad
+
+import ViewAdapter
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.domain.Order
+
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
```

Comment:
What is ARG_PARAM1?
It seems like default constants when you create the blank fragment.
at least change those variable names with proper ones.